app/Http/Controllers/Api/ReviewsController.php

```diff
@@ -55,48 +55,19 @@ public function __construct(PostService $posts, ReportbackService $reportbacks)
     /**
      * Update a post(s)'s status when reviewed.
      *
-     * @param \Illuminate\Http\Request $request
+     * @param Rogue\Http\Requests\ReviewsRequest $request
      *
      * @return \Illuminate\Http\Response
      */
-    public function reviews(Request $request)
+    public function reviews(ReviewsRequest $request)
     {
-        $this->validate($request, [
-            '*.rogue_event_id' => 'required',
-            '*.status' => 'required',
-        ]);
```

Comment:
@sbsmith86 @katiecrane moved this out to it's own `ReviewsRequest.php` since we aren't dealing with multiple reportback items anymore. 

@sbsmith86 I got it to work (you were right, my headers were wrong in Postman!) to return a `500` response code. However, the message is not filled in automatically like it is when I don't follow validation rules in PostRequest. I also didn't follow validation rules in the ReactionRequest and no message is returned there either, just a `500` code. Is this ok to just return a `500` or should it be also returning the message?